cypress/e2e/citation-plugin.cy.js

```diff
@@ -0,0 +1,36 @@
+describe('main', () => {
+  it('the citation modal should load correctly', () => {
+    cy.viewport(1280, 720);
+
+    cy.visit('test.html');
+    cy.get('.say-editor-hints__insert').contains('button', 'Insert').click();
+    cy.get('.say-editor-hints__insert')
+      .contains('button', 'Insert reference')
+      .click();
+    cy.get('.citaten--main-container')
+      .find('.au-c-main-container__content')
+      .find('ul')
+      .as('list');
```

Comment:
I thought the idea of `as()` was to mark elements for use between functions, for example when running set-up code. I think in this case we can just store the result of `find()` in a const and then check that.

Reply:
Yeah, I thought so too, but apparently it is an anti-pattern: https://docs.cypress.io/guides/references/best-practices#Assigning-Return-Values. As cypress works in an asynchronous way.

Reply:
Ah, good to know. Not sure where I got that impression from.